### PR TITLE
Update meme.py with .lick meme command

### PR DESCRIPTION
**Pull request by Aceofgods**
Never done this before.

### CODE REVIEW

File: robocop_ng/cogs/meme.py [hunk 1]
```diff
@@ -56,6 +56,19 @@ async def warm_member(self, ctx, user: Optional[discord.Member]):
                 f"({fahrenheit}°F, {kelvin}K)."
             )
 
+    @commands.check(check_if_staff_or_ot)
+    @commands.command(hidden=True, name="lick")
+    async def licky(self, ctx, user: Optional[discord.Member]):
```
Comment (TSRBerry) [inline robocop_ng/cogs/meme.py:61]:
```suggestion
    @commands.command(hidden=True)
    async def lick(self, ctx, user: Optional[discord.Member]):
```
Just to simplify the syntax a little bit.

**Review by TSRBerry — APPROVED** [review]
Lgtm! :p

**Comment by marysaka** [comment]
Why do we want this? :thinking:

**Comment by TSRBerry** [comment]
Memes are fun and I think it's hilarious that this is their first PR on GitHub ever!

**Review by marysaka — APPROVED** [review]

**Pull request merged**